include/nbl/asset/filters/CBlitImageFilter.h

```diff
@@ -78,47 +76,58 @@ class CBlitImageFilterBase : public impl::CSwizzleableAndDitherableFilterBase<Sw
 };
 
 // copy while filtering the input into the output, a rare filter where the input and output extents can be different, still works one mip level at a time
-template<typename Swizzle=DefaultSwizzle, typename Dither=CWhiteNoiseDither, typename Normalization=void, bool Clamp=true, class KernelX=CBoxImageFilterKernel, class KernelY=KernelX, class KernelZ=KernelX, typename lut_value_type = typename KernelX::value_type>
-class CBlitImageFilter : public CImageFilter<CBlitImageFilter<Swizzle,Dither,Normalization,Clamp,KernelX,KernelX,KernelX>>, public CBlitImageFilterBase<typename KernelY::value_type,Swizzle,Dither,Normalization,Clamp>
+template<
+	typename Swizzle				= DefaultSwizzle,
+	typename Dither					= CWhiteNoiseDither,
+	typename Normalization			= void,
+	bool Clamp						= true,
+	typename BlitUtilities			= CBlitUtilities<>,
+	typename LutDataType			= float>
```

Comment:
eh, lets stick the LUT data type into `CBlitUtilities`, will de-template a lot of member functions.